src/components/AccountDrawer/SettingsMenu.tsx

```diff
@@ -53,28 +34,77 @@ const ToggleWrapper = styled.div`
   margin-bottom: 24px;
 `
 
-export default function SettingsMenu({ onClose }: { onClose: () => void }) {
+const SettingsButtonWrapper = styled(Row)`
+  ${ClickableStyle}
+`
+
+const StyledChevron = styled(ChevronRight)`
+  color: ${({ theme }) => theme.textSecondary};
+`
+
+const SettingsButton = ({
+  title,
+  currentState,
+  onClick,
+  testId,
+}: {
+  title: ReactNode
+  currentState: ReactNode
+  onClick: () => void
+  testId?: string
+}) => (
+  <SettingsButtonWrapper data-testid={testId} align="center" justify="space-between" onClick={onClick}>
+    <ThemedText.SubHeaderSmall color="textPrimary">{title}</ThemedText.SubHeaderSmall>
+    <Row gap="xs" align="center" width="min-content">
+      <ThemedText.LabelMedium color="textPrimary">{currentState}</ThemedText.LabelMedium>
```

Comment:
some languages wrap in menu due to min-content
![Screenshot 2023-08-17 at 12 04 51 PM](https://github.com/Uniswap/interface/assets/11512321/3e5252bb-96c0-4bce-99d9-df7d682b16e8)

Reply:
nice find